src/application/BoDCharacter.java

```diff
@@ -0,0 +1,53 @@
+package application;
+import java.awt.Graphics2D;
+import java.awt.event.KeyEvent;
+import java.util.HashSet;
+
+public class BoDCharacter extends GameObject
+{
+    HashSet<Integer> movementKeys;
+	public BoDCharacter(Graphics2D g2d)
+	{
+	    movementKeys = new HashSet<>();
+	    movementKeys.add(Keys.A);
+        movementKeys.add(Keys.S);
+        movementKeys.add(Keys.D);
+        movementKeys.add(Keys.W);
+        movementKeys.add(Keys.UP);
+        movementKeys.add(Keys.DOWN);
+        movementKeys.add(Keys.LEFT);
+        movementKeys.add(Keys.RIGHT);
+        
+		this.physics = new Physics(this);
+		this.body = new Body(this);
+		this.view = new View(this, g2d);
+	}
+	public void KeyPressed(KeyEvent e)
```

Comment:
this code doesnt support pressing more than one key at any point in time. it will not be possible to move diagonally.